src/js/post-911-gib-status/components/UserInfoSection.jsx

```diff
@@ -20,7 +20,7 @@ class UserInfoSection extends React.Component {
       currentAsOfAlert = (
         <div className="usa-alert usa-alert-info">
           <div className="usa-alert-body">
-            <h4 className="usa-alert-heading">This information is current as of {todayFormatted}</h4>
+            <h4 id="up-to-date-header" className="usa-alert-heading">This information is current as of {todayFormatted}</h4>
```

Comment:
I played around with fancy selectors for over an hour before I decided that an `id` is probably simpler and more readable.